packages/origin-247-certificate/src/offchain-certificate/validators/decorators/is-energy-value.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    registerDecorator,
+    ValidationArguments,
+    ValidationOptions,
+    ValidatorConstraint,
+    ValidatorConstraintInterface
+} from 'class-validator';
+import { BigNumber } from 'ethers';
+
+@ValidatorConstraint({ name: 'isEnergyValue' })
+export class IsEnergyValueConstraint implements ValidatorConstraintInterface {
+    public validate(value: string) {
+        const isNegative = BigNumber.from(value).isNegative();
```

Comment:
if we send something that is not proper BigNumber (float, random string) validator will fail without returning any value, so I'm not sure if this will work correctly

Reply:
fixed